Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteAboutController.java

```diff
@@ -0,0 +1,43 @@
+package edu.asu.spring.quadriga.web.publicwebsite;
+
+import java.security.Principal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+
+@Controller
+public class WebsiteAboutController {
+
+	@Autowired
+	private IRetrieveProjectManager projectManager;
+
+	private IProject getProjectDetails(String name) throws QuadrigaStorageException {
```

Comment:
I don't think you need this method, since it's just one line of code, just call the projectManager directly in your method (line 36).